rv-monitor/src/main/java/com/runtimeverification/rvmonitor/java/rvj/Main.java

```diff
@@ -302,36 +304,29 @@ public static void process(String arg) throws RVMException {
      * RV-Monitor.
      */
     public static void print_help_ext() {
-        System.out
-        .println("Usage: java [-cp rv_monitor_classpath] com.runtimeverification.rvmonitor.java.rvj.Main [-options] files");
+        System.out.println("Usage: java [-cp rv_monitor_classpath] com.runtimeverification.rvmonitor.java.rvj.Main [-options] files");
         System.out.println("");
-        System.out
-        .println(" Options enabled by default are prefixed with \'+\'");
+        System.out.println(" Options enabled by default are prefixed with \'+\'");
         System.out.println("    -h --help\t\t\t  print this help message");
-        System.out
-        .println("    --version\t\t\t  display RV-Monitor version information");
+        System.out.println("    --version\t\t\t  display RV-Monitor version information");
         System.out.println("    -v | --verbose\t\t  enable verbose output");
         System.out.println("    --debug\t\t\t  enable verbose error message");
         System.out.println();
 
-        System.out
-        .println("    -d <output path>\t\t  select directory to store output files");
-        System.out
-        .println("    -n <name>\t\t\t  use the given class name instead of source code name");
+        System.out.println("    -d <output path>\t\t  select directory to store output files");
+        System.out.println("    -n <name>\t\t\t  use the given class name instead of source code name");
         System.out.println();
 
-        System.out
-        .println("    -s | --statistics\t\t  generate monitor with statistics");
-        System.out
-        .println("    --noopt1\t\t\t  don't use the enable set optimization");
+        System.out.println("    -s | --statistics\t\t  generate monitor with statistics");
+        System.out.println("    --noopt1\t\t\t  don't use the enable set optimization");
         System.out.println();
 
-        System.out
-        .println("    --finegrainedlock\t\t  use fine-grained lock for internal data structure");
-        System.out
-        .println("    --weakrefinterning\t\t  use WeakReference interning in indexing trees");
+        System.out.println("    --finegrainedlock\t\t  use fine-grained lock for internal data structure");
+        System.out.println("    --weakrefinterning\t\t  use WeakReference interning in indexing trees");
         System.out.println();
 
+        System.out.println("    --controlAPI\t\t  generate extra APIs for enable/disable monitors");
```

Comment:
What does this `controlAPI` option do and when does one need to use it? It would be good to have these documented.